sast-terraform/README.md

```diff
@@ -0,0 +1,39 @@
+# SAST Terraform CircleCI orb
+
+This orb can be used to run the static code analysis tools via the CLI against a target directory containing Terraform infrastructure-as-code.
```

Comment:
Maybe worth also adding a little bit about the baseline file and how to generate it to ease adoption for teams?